packages/web3torrent/src/logger.ts

```diff
@@ -28,15 +29,72 @@ const getCircularReplacer = () => {
     return value;
   };
 };
+class LogBlob {
+  private parts = [];
+  private _blob?: Blob;
 
-const browser =
-  LOG_TO_FILE && IS_BROWSER_CONTEXT
-    ? {write: o => console.log(JSON.stringify({...o, name}, getCircularReplacer()))}
-    : undefined;
+  append(part) {
+    this.parts.push(part);
+    this._blob = undefined;
+  }
+
+  get blob() {
+    if (!this._blob) this._blob = new Blob(this.parts, {type: 'text/plain'});
+
+    return this._blob;
+  }
+}
+
+const logBlob = new LogBlob();
+
+const serializeLogEvent = o => JSON.stringify({...o, name}, getCircularReplacer());
+
+let browser: any = IS_BROWSER_CONTEXT
+  ? {
+      transmit: {
```

Comment:
What calls `transmit`?

Reply:
Transmit is a configuration option of pino's browser API. The `send` callback is called each time the Pino logger is writing a log.

If you were to hook Pino up to data dog or something, you could fire off an HTTP request in this callback as well.